### PR TITLE
Remove token list from NFT

### PR DESCRIPTION
**Pull request by RafilxTenfen**
### Removed the token list from NFT type

Add verification for `TokenList` 

### Description

- Add `v-if` verification of global state actual token type

### How to Test

- Config your `.env` file like this
```env
...
VUE_APP_NFT_AVAILABLE=true
```

- Run the server

#### Case 1

1. Run the server
```shell
$~ npm run serve
```

2. Click on NFT button

__Expected Result__
- The token list should dissapear

### CODE REVIEW

File: src/components/formWrapper/FormWrapper.vue [hunk 1]
```diff
@@ -70,15 +70,15 @@ export default {
   },
   mounted() {
     this.currentTab = formWrapperTabs[0]
-    this.globalStore.tokenTypeSelected = this.currentTab.id
+    this.globalStore.actualTokenType = this.currentTab.tokenType
```
Comment (marcciosilva) [inline src/components/formWrapper/FormWrapper.vue:73]: I'd change 'actual' for 'current'

File: src/components/formWrapper/formWrapperTabs.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
   {
-    id: 'erc20',
+    tokenType: TOKEN_TYPE_ERC_20,
     label: 'ERC20 Tokens',
     component: CrossForm,
   },
   {
-    id: 'erc721',
+    tokenType: TOKEN_TYPE_ERC_721,
     label: 'NFT',
```
Comment (marcciosilva) [inline src/components/formWrapper/formWrapperTabs.js:16]: maybe 'ERC721 NFT' for this to be extensible in the future? (and also so users know that we only support this standard for now)

Comment (RafilxTenfen) [reply src/components/formWrapper/formWrapperTabs.js]: done at https://github.com/rsksmart/tokenbridge-ui/pull/52/commits/9a00c8c01e807ba45a300cfc75b64c45df26b3f9

File: src/components/formWrapper/FormWrapper.vue (hunk 1 above)
Comment (RafilxTenfen) [reply src/components/formWrapper/FormWrapper.vue]: done at https://github.com/rsksmart/tokenbridge-ui/pull/52/commits/71e1d66d07a4e1ec59e8323e2504f1d9caa3465d

**Pull request merged**